LykkeApi2/Modules/Api2Module.cs

```diff
@@ -126,7 +126,7 @@ private void BindServices(ContainerBuilder builder, BaseSettings settings)
                 .SingleInstance();
 
             builder.RegisterInstance(settings.SessionCheck);
-            builder.RegisterInstance(settings.PrivateWallet ?? new PrivateWalletSettings());
+            builder.RegisterInstance(settings.PrivateWalletses ?? new PrivateWalletsSettings());
```

Comment:
Likely, rider renamed to `PrivateWalletses` automatically, but I think the regular plural form will do just fine. Hobbitses keep their precious in the walletses.

Reply:
ah true